ICAFIX/hcp_fix_multi_run

```diff
@@ -838,86 +872,156 @@ fi
+		TrainingData="${FSL_FIXDIR}"/training_files/HCP_Style_Single_Multirun_Dedrift.RData
+		if [[ ! -f "$TrainingData" ]]; then
+			log_Err_Abort "default training file '$TrainingData' not found, please update your fix installation"
+		fi
 	fi
-	
-	# if the specified TrainingData is not a full path to an existing file,
-	# assume that the user is specifying the name of a file in the training_files folder in FSL_FIXDIR
-	if [ ! -f "${TrainingData}" ]; then 
-		TrainingData=${FSL_FIXDIR}/training_files/${TrainingData}
+
+	## MPH: The 'fix' script itself will continue to log to its own custom files
+	## Alert user to where those are
+	log_Msg "Check ${concatfmrihp}.ica/fix/logMatlab.txt for log output from feature extraction"
+	log_Msg "Check ${concatfmrihp}.ica/.fix_2b_predict.log for log output from component classification"
+	log_Msg "Check ${concatfmrihp}.ica/.fix.log for log output from cleanup stage"
+
+else
+	log_Msg "Running pyFIX"
+
+	# Get concatenated Movement_Regressors_demean.txt into the format expected by pyfix
+	if [ -f Movement_Regressors_demean.txt ] ; then
+		log_Debug_Msg "About to create ${concatfmrihp}.ica/mc/prefiltered_func_data_mcf.par file"
+		cat Movement_Regressors_demean.txt | awk '{ print $4 " " $5 " " $6 " " $1 " " $2 " " $3}' > ${concatfmrihp}.ica/mc/prefiltered_func_data_mcf.par
+	else
+		log_Err_Abort "Movement_Regressors.txt not retrieved properly." 
 	fi
-	
-	# finally, if the TrainingData file is not found, report an error and get out of here
-	if [[ ! -f "${TrainingData}" ]]; then
-		log_Err_Abort "FIX training data not found: ${TrainingData}"
+
+	# if specified, copy template brain for creation of masks
+	if [[ ${T1wTemplateBrain} ]]; then
+		$FSLDIR/bin/imcp ${T1wTemplateBrain} ${concatfmrihp}.ica/reg/standard
 	fi
-else
-	# User has not specified a training data file
-	log_Msg "training data file not specified"
-	TrainingData="${FSL_FIXDIR}"/training_files/HCP_Style_Single_Multirun_Dedrift.RData
-	if [[ ! -f "$TrainingData" ]]; then
-		log_Err_Abort "default training file '$TrainingData' not found, please update your fix installation"
+
+	# Handle user specified training data file
+	if [[ "${TrainingData}" != "" ]]; then
+		# User has specified a training data file
+		
+		# if training data is in legacy RData format, and not a built-in model, convert it
+		builtInModels="@HCP25_hp2000@HCP7T_hp2000@HCP_Style_Single_Multirun_Dedrift@NHP_HCP_Macaque@NHP_HCP_MacaqueCyno@Standard@UKBiobank@WhII_MB6@WhII_Standard@"
+		if [[ "${TrainingData}" == *.RData ]]; then 
+			TrainingData=${TrainingData%.RData}
+			if [[ ! ${builtInModels} == *"@${TrainingData}@"* ]]; then 
+				${FSL_FIXDIR}/convert_fix_model ${TrainingData}.RData ${TrainingData}.pyfix_model
+				TrainingData=${TrainingData}.pyfix_model
+			fi
+		fi
+	else
+		# User has not specified a training data file
+		log_Msg "training data file not specified, setting to HCP_Style_Single_Multirun_Dedrift"
+		TrainingData=HCP_Style_Single_Multirun_Dedrift
```

Comment:
This specification here is only a relative path, not a full path.  Also, no extension/suffix is specified.  Also, does `FSL` come with a pre-computed version of the `.pyfix_model` form of this?  If not, don't we need to run a conversion on it once we've finally settled on which version of the training data we want to use?

Reply:
I believe they provided all the ones that legacy fix did already converted.

Reply:
So, maybe conversion isn't necessary here, but still seems likely that the pre-converted version supplied by FSL won't be found with the current code.  Don't we need to set an absolute path to the TrainingData here?  [Something like `TrainingData=${FSL_FIXDIR}/${TrainingData}` (or wherever the FSL distro is placing the pre-converted models)?]

Reply:
Also, can we add a sanity check to the end of this section that the `.pyfix_model` version of TrainingData actually exists?